routes/users.js

```diff
@@ -23,5 +23,6 @@ router.post("/picture", authenticate, upload.single("profile"), users.postUserPi
 router.patch("/profileURL", authenticate, userValidator.updateProfileURL, users.profileURL);
 router.patch("/rejectDiff", authenticate, authorizeRoles([SUPERUSER]), users.rejectProfileDiff);
 router.patch("/:userId", authenticate, authorizeRoles([SUPERUSER]), users.updateUser);
+router.get("/suggestedUsers/:skillId", authenticate, authorizeRoles([SUPERUSER]), users.getSuggestedUsers);
```

Comment:
NIT comment: It would be better if you could de-structure all the controller functions of `users` instead of using `users.*`